code/src/components/app-range-slider/AppRangeSlider.test.tsx

```diff
@@ -9,13 +9,29 @@ const mockHandleChange = jest.fn();
 
 describe("AppRangeSlider", () => {
   describe("with default props", () => {
-    test("uses default props correctly", () => {
+    test("uses default props correctly", async () => {
       renderWithProviders(<AppRangeSlider />);
 
       const rangeStartInput = screen.getByTestId("range-start");
       expect(rangeStartInput).toHaveAttribute("min", "0");
       expect(rangeStartInput).toHaveAttribute("max", "20000");
       expect(rangeStartInput).toHaveAttribute("step", "10");
+
+      const rangeEndInput = screen.getByTestId("range-end");
+      await typeIntoInput(rangeEndInput, 90);
```

Comment:
Maybe we need to add some assertion here?

Reply:
That test was wrtten to pass mutation that removed `?` in this line `onChange?.(typedEvent, [limitedNewValue, rangeEnd]);`, so it was working. I added simple assertion for it to be more understandable.